tests/Makefile.am

```diff
@@ -46,6 +46,7 @@ testlayers_SOURCES = \
     ./common/layerrenderer.cpp \
     ./common/gllayerrenderer.cpp \
     ./common/glcubelayerrenderer.cpp \
+	./common/videolayerrenderer.cpp \
```

Comment:
Is this change supposed to be part of another pull request?  Suggest combining it with the corresponding commit.

Reply:
Yes, there are a couple of patches need to be reviewed in parallel and there are some commit order dependence among them. The list of PRs for these patches have been committed into my own git repo:
https://github.com/shuangwan01/IA-Hardware-Composer/pulls

Not sure if there is an efficient way to push PRs into this project directly and manage the further revisions accordingly.

Reply:
Is it better to combine the patches under one commit?  The specific commit order requirement may be difficult to maintain because only the author and reviewers are familiar with it right? :)

Reply:
Yes, agree with you that one commit will make the PR maintenance more easier. But this feature is involving 3 authors and feature itself even doesn't have tested under Android. So it's expected large number of revisions according to real tests/debugs and review comments. Breaking down into small code pieces will make this practice more efficient and is also easier for community to conduct code review :-)

Reply:
Sounds appropriate :) 👍